src/ResourceManager/Compute/Commands.Compute/Commands.Compute.csproj

```diff
@@ -82,10 +82,22 @@
       <HintPath>..\..\..\packages\Microsoft.Data.Services.Client.5.8.2\lib\net40\Microsoft.Data.Services.Client.dll</HintPath>
       <Private>True</Private>
     </Reference>
+    <Reference Include="Microsoft.Rest.ClientRuntime, Version=2.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
@EvgenyAgafonchikov you shouldn't need to add these references in your csproj file, they should be derived from [`Common.Dependencies.targets`](https://github.com/Azure/azure-powershell/blob/preview/tools/Common.Dependencies.targets)